handbook/engineering/developer-tools/index.md

```diff
@@ -0,0 +1,15 @@
+# Developer tools team
```

Comment:
I think the right way to document this is not to document a new team that doesn't exist, but instead add a goal to the distribution team and document this in the distribution team's growth plan (e.g. we will grow this team and plan on spinning off a dev tools team in the future.

Examples:
https://about.sourcegraph.com/handbook/engineering/web#growth-plan
https://about.sourcegraph.com/handbook/engineering/cloud#growth-plan

Reply:
I can merge this back to distribution, but I was actually using what we did in https://github.com/sourcegraph/about/pull/1150/ for the "Backend infrastructure team" as an example.

Reply:
Ah, I didn't realize we still had that (we had some other prospective teams that I deleted and merged back into existing teams).

You can leave this if you want, but also update the distribution team to reference the growth plan.

Reply:
I would like to keep it as an initial idea of that team's vision and goals, but I will add this to distribution's a distribution growth plan so its reflected there.